package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "reactmentoring",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "server": "node server/app.js",
+    "build": "webpack --mode production",
```

Comment:
let's create two scripts for prod and dev builds

Reply:
done